content/applications/marketing/marketing_automation.rst

```diff
@@ -1,121 +1,201 @@
+from the main Odoo dashboard, to open the :guilabel:`Campaigns` dashboard, which displays six
+campaign template cards:
+
+.. image:: marketing_automation/campaigns-dashboard.png
    :align: center
-   :alt: Match records with all or any of the rules in Filter field for marketing campaigns.
+   :alt: Six campaign template cards on the Campaigns dashboard of the Marketing Automation app.
 
-When the first field of the rule equation is clicked, a nested drop-down menu of options appears on
-the screen where specific criteria is chosen based on needs of the campaign.
+- | :guilabel:`Tag Hot Contacts`
+  | :guilabel:`Send a welcome email to contacts and tag them if they click it.`
+- | :guilabel:`Welcome Flow`
+  | :guilabel:`Send a welcome email to new subscribers, remove the address that bounced.`
+- | :guilabel:`Double Opt-in`
+  | :guilabel:`Send an email to new recipients to confirm their consent.`
+- | :guilabel:`Commercial prospection`
+  | :guilabel:`Send a free catalog and follow-up according to reactions.`
+- | :guilabel:`Schedule Calls`
+  | :guilabel:`If a lead is created for existing contact, schedule a call with their salesperson.`
+- | :guilabel:`Prioritize Hot leads`
+  | :guilabel:`Send an email to new leads and assign them a high priority if they open it.`
 
-The remaining fields on the rule equation further define the criteria, which is used to determine
-which records in the database to include or exclude in the execution of the campaign.
+These templates are designed to be used as starting points for creating new campaigns. Click one of
+the template cards to open the campaign form.
 
-To add another rule, either click the :guilabel:`➕ (plus sign)` icon to the right of the filtering
-rule, or click :guilabel:`New Rule` beneath the rule equation fields. When either are clicked, a new
-series of rule fields appears.
+.. tip::
+   To display the campaign template cards again after a campaign has been created, type the name of
+   a campaign that does **not** exist in the database into the :guilabel:`Search...` bar, then press
+   :kbd:`Enter`.
 
-To add a branch of multiple rules at the same time, click the :guilabel:`branch` icon, located to
-the right of the :guilabel:`➕ (plus sign)` icon. When clicked, two additional sub-rule equation
-fields appear beneath the initial rule.
+   For example, searching for `empty` displays the campaign template cards again, as long as there
+   is not a campaign with the name "empty" in the database.
 
-.. image:: marketing_automation/rule-branch-filter-sample.png
-   :align: center
-   :alt: Sample of how the rule branches look in the filter section of a marketing campaign.
+Targets and filters
+===================
+
+On the campaign form, the :guilabel:`Target` and :guilabel:`Filter` section, also referred to as the
+domain, contains the fields used to define the target audience for the campaign's reach (i.e.,
+records).
+
+Records
```

Comment:
In this section you could also probably mention how there are different types of records depending on the **Target** model that's selected, e.g. Leads/Opportunity, Event Registration, etc.